### PR TITLE
Add support for tilt on multiple huion/gaomon tablets

### PR DESCRIPTION
**Pull request by jamesbt365**

### CODE REVIEW

File: TABLETS.md [hunk 1]
```diff
@@ -103,7 +108,6 @@
 | XP-Pen Star G640              |     Supported     |
 | XP-Pen Star G640 V2           |     Supported     |
 | XP-Pen Star G640S             |     Supported     |
-| XP-Pen Star G960              |     Supported     |
```
Comment (X9VoiD) [inline TABLETS.md:106]: Undo this change

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: TABLETS.md (hunk 1 above)
Comment (jamesbt365) [reply TABLETS.md]: bruh

**Review by X9VoiD — APPROVED** [review]
